XCTest.xcodeproj/project.pbxproj

```diff
@@ -409,13 +407,22 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				INFOPLIST_FILE = Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				MACOSX_DEPLOYMENT_TARGET = 10.11;
 				MTL_ENABLE_DEBUG_INFO = YES;
 				ONLY_ACTIVE_ARCH = YES;
-				SDKROOT = macosx;
```

Comment:
Very good question!

Please note that most of the changes I made are based on [PromiseKit](https://github.com/mxcl/PromiseKit) and the [article](http://promisekit.org/news/2016/08/Multiplatform-Single-Scheme-Xcode-Projects/).

We also should note that we're making the multi-platform single-scheme framework now, so each platform needs it's own `SDKROOT`.

I tried the same setup for `SDKROOT` as for `PromiseKit` (just by removing `SDKROOT` on project level) and it works for `swift-corelibs-xctest`. But I had to add to specify `SDKROOT` for `SwiftXCTestFunctionalTests` target, because without `SDKROOT` it failed to build.

**PS**. I agree that it would be better having `SDKROOT` on project level for each platform, but Xcode only allows configuring it for architecture, not for platform.

It would be awesome if you provide any better solution. Thanks!